docs/_includes/footer.html

```diff
@@ -11,28 +11,28 @@
                 </ul>
             </div>
             <div class="col-md-3">
-                <ul>PRODUCTS
+                <ul><div class="footer-title">Products</div>
                     <li><a href="https://insomnia.rest/products/insomnia">Insomnia</a></li>
                     <li><a href="https://insomnia.rest/products/inso">Inso (CLI)</a></li>
                 </ul>
             </div>
             <div class="col-md-3">
-                <ul>RESOURCES
+                <ul><div class="footer-title">Resources</div>
                     <li><a href="https://app.insomnia.rest/app/login/">Sign In</a></li>
                     <li><a href="#">Documentation</a></li>
                     <li><a href="https://insomnia.rest/support">Support</a></li>
                 </ul>
             </div>
             <div class="col-md-3">
-                <ul>LEGAL
+                <ul><div class="footer-title">Legal</div>
                     <li><a href="https://insomnia.rest/privacy">Privacy Policy</a></li>
                     <li><a href="https://insomnia.rest/terms">Terms & Conditions</a></li>
                 </ul>
             </div>
         </div>
         <div class="row align-items-md-stretch">
             <div class="col-md-12">
-                <small id="footer-copyright">© Kong Inc. 2021</small> 
+                <small id="footer-copyright">© Kong Inc. 2022</small> 
```

Comment:
Looks like we need to bump this on the marketing website too

Reply:
there's a PR for this on the marketing side to make it dynamic.  we could do the same thing here if we wanted https://github.com/Kong/insomnia-website/pull/21
```suggestion
                <small id="footer-copyright">© Kong Inc. {new Date().getFullYear()}</small> 
```